docs/sparkr.md

```diff
@@ -210,6 +210,43 @@ head(df)
 {% endhighlight %}
 </div>
 
+### Model Formulae
```

Comment:
+1. I think calling the section `Machine Learning` should be good. Since we only have `glm` right now, we can probably get by without a subsection right now.